Added new benchmark among-us with one problem file

katin139 opened the pull request:
This is our original submission with the domain and one problem file. We will start creating the different problem files and adding them here!

katin139 committed:
Added new benchmark among-us with one problem file

This is our original submission with the domain and problem file.

haz commented:
Awesome, thanks! The important thing is that we have a decent spread of difficulty (from easy to near impossible to solve). I can help provide some guidance on where each problem fits for that.

katin139 committed:
Created 10 Easy Problem Sets

These are 10 problem sets that are able to run without receiving a pending error from the remote server. NFC - not fully connected
1p - 1 person
nr - n number of rooms

katin139 commented:
We have added 10 easy problem sets. We based it on what the remote server could solve without receiving a pending error. Let us know if these are all okay & we will start on some medium and hard problems(:
